### ISSUE TITLE
Explain Functioning or Login

### ISSUE THREAD

**Issue by SystematicError** [ifvictr/nitrotype.js#8]

**Comment by SystematicError** [comment]
I know you also have to generate a uhash, can you also explain that please :)

**Comment by Ray-Adams** [comment]
@SystematicError I'm not experienced with Python, but I believe I can assist you with your second inquiry. 

The uhash can be obtained by retrieving the value of the `ntuserrem` cookie. If you are trying to generate the uhash yourself, this may be why you are seeing the `{"success":false,"data":{"message":"Invalid access method"}}` response.

**Comment by SystematicError** [comment]
Doesnt the cookie get generated after login?

**Comment by ghost** [comment]
Yes, the cookie gets generated after the login, as the uhash isn't required for logging in.

**Comment by SystematicError** [comment]
Oh ok, other than uhash what else is required for login?

**Comment by mouncg** [comment]
lmao, it  isn't hard

**Comment by mouncg** [comment]
Symantec, why do you still exist lmao

**Comment by SystematicError** [comment]
Can you explain if it's easy :)

**Comment by mouncg** [comment]
basically, using the api to login is almost impossible now, NT are changing it lmao travis told me :P

**Comment by Pool800** [comment]
Wait so just out of interest is there an account checker for nitrotype?
